aws/resource_aws_s3control_bucket.go

```diff
@@ -91,8 +96,8 @@ func resourceAwsS3ControlBucketCreate(d *schema.ResourceData, meta interface{})
 
 	d.SetId(aws.StringValue(output.BucketArn))
 
-	if v := d.Get("tags").(map[string]interface{}); len(v) > 0 {
-		if err := keyvaluetags.S3controlBucketUpdateTags(conn, d.Id(), nil, v); err != nil {
+	if len(tags) > 0 {
+		if err := keyvaluetags.S3controlBucketUpdateTags(conn, d.Id(), nil, tags); err != nil {
```

Comment:
i _think_ here we'll want to pass in the map `tags.Map()` since the `S3controlBucketUpdateTags` internally calls `New` on the tags value passed in